### PR TITLE
Fix minor issues with the docs/tutorial

### PR DESCRIPTION
**Pull request by Avataw**
## ✅ What it does

- Readme
  - added small section to clarify where to make changes to the docs and where to find them
 
- Chapter Two > Routing Params
  - fixed wrong type in ArticleCell.tsx
  - added missing update of ArticleCell.tsx in the end

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -12,6 +12,12 @@ yarn install
 yarn start
 ```
 
+#### Making Changes
+Changes should be made in the `./docs` directory and not in the generated `versioned_docs` or `versioned_sidebars` directly.
```
Comment (Tobbe) [inline docs/README.md:16]:
```suggestion
Changes should be made in the `./docs` directory and not in the generated `versioned_docs` or `versioned_sidebars` directories directly.
```

File: docs/README.md [hunk 2]
```diff
@@ -12,6 +12,12 @@ yarn install
 yarn start
 ```
 
+#### Making Changes
+Changes should be made in the `./docs` directory and not in the generated `versioned_docs` or `versioned_sidebars` directories directly.
+
+After the ```yarn start``` command ran successfully you should be able to see your changes in the local [Canary Version](http://localhost:3000/docs/canary/index)
+
+
```
Comment (jtoar) [inline docs/README.md:20]: Thanks for this! I definitely spent at least 10 minutes refreshing `localhost:3000 trying` to understand why the change wasn't showing up. 🤦 The suggestions below are mostly just stylistic:

```suggestion
#### Making Changes

Changes should be made in the `./docs` directory and not in the generated `./versioned_docs` or `./versioned_sidebars` directories directly.

After running `yarn start`, you should be able to see your changes in the local [Canary Version](http://localhost:3000/docs/canary/index).

```

**Comment by jtoar** [comment]
Thanks @Avataw! Looks good to me. @cannikin was there a reason you left the codeblock updating `ArticleCell` (singular) out? Was it just to keep things short since it's almost the same as updating `ArticlesCell` (plural)?

File: docs/README.md (hunk 2 above)
Comment (Avataw) [reply docs/README.md:20]: Me too :D That's why I added it hahaha :D

**Comment by Tobbe** [comment]
LGTM

**Review by cannikin — APPROVED** [review]

**Comment by cannikin** [comment]
> @cannikin was there a reason you left the codeblock updating `ArticleCell` (singular) out?

Probably just overlooked it, thanks for the fix!
